### DISCUSSION TITLE
Are email addresses publicized when uploading a package?

### DISCUSSION THREAD

**Discussion by mybearworld** [jsr-io/jsr#532] [Q&A] [answered]
![Resources on your account: Email addresses (read) - View your email addresses](https://github.com/jsr-io/jsr/assets/130385691/0c8dd878-16a5-4bbc-8002-269d918d28cf)

Will my email address be publicized when I upload a package, or is it only used to send out emails?

**Comment by marvinhagemeister** [comment, marked as answer]
No, email addresses are not made public during publishing. I don't think they are used at all during the publishing process as the only thing we care about is authentication and that happens via GitHub already.

Email addresses are only used to send emails so that users can accept invites to scopes from someone else and stuff like that. I'm not aware of any usage for emails outside of that.

**Reply by mybearworld** [reply]
Great, thank you!
